relay/sources/apache/rudder-apache-relay-ssl.conf

```diff
@@ -44,10 +44,14 @@ Alias /inventories /var/rudder/inventories/incoming
   AuthName "WebDAV Storage"
   AuthType Basic
   AuthUserFile /opt/rudder/etc/htpasswd-webdav-initial
-  Require valid-user
 
-  # rudder-networks-24.conf is automatically generated according to the hosts allowed by rudder.
-  Include /opt/rudder/etc/rudder-networks-24.conf
+  <RequireAll>
+    Require valid-user
+    <RequireAny>
```

Comment:
why any with only one entry ?

Reply:
because the included file contains a list of require

Reply:
oh yes